ci/docker/install/android_arm64_openblas.sh

```diff
@@ -25,8 +25,41 @@ pushd .
 git clone https://github.com/xianyi/OpenBLAS.git
 cd OpenBLAS
 make -j$(nproc) TARGET=ARMV8 ARM_SOFTFP_ABI=1 HOSTCC=gcc NOFORTRAN=1 libs
-# Can't be run (utility not compiled for the target platform)
-#make install
-cp *.h /usr/include
-cp libopenblas.a /usr/local/lib
+
+# Ideally, a simple `make install` would do the job. However, OpenBLAS fails when running `make
```

Comment:
Actually, you can set the [`PREFIX`](https://github.com/apache/incubator-mxnet/blob/master/ci/docker/install/arm_openblas.sh#L26) instead.

Reply:
Yes, but the problem is that the utility `getarch` which is invoked by the `Makefile` fails to execute. That would be another rabbit hole that I didn't want to get into.